plugin.xml

```diff
@@ -40,14 +40,14 @@
       <preference name="BackupWebStorage" value="local" />
 
       <!-- To allow XHR requests with the new Whitelist plugin -->
-      <allow-navigation href="https://localhost" />
-      <allow-navigation href="https://*.force.com" />
-      <allow-navigation href="https://*.salesforce.com" />
+      <allow-navigation href="https://localhost" subdomains="true" requires-forward-secrecy="false" />
```

Comment:
This is the fix. `Cordova` added new attributes here for ATS settings and `requires-forward-secrecy` defaults to `true` by default, but our servers need them to be `false`.